cluster/up.sh

```diff
@@ -33,5 +33,6 @@ for node in $(./cluster/kubectl.sh get nodes --no-headers | awk '{print $1}'); d
 done
 
 echo 'Deploying multus'
+# Taking latest multus (image is breathing and its on docker.io)
```

Comment:
Please advice, do you want me to snapshot the yaml and change the image there to an image that we upload to quay?
or leave it as is, it will be breathing
(we might need to update the job to use proxy, but this is little other story)

Reply:
@phoracek @EdDev 
as part of the effort to move images from docker.io to quay on ovs-cni (this PR),
please suggest if we want to freeze a specific multus yaml and push its image to quay,
currently it always fetch the latest yaml + latest stable version, and its hosted on docker.io
so we must either leave it as is, or freeze both yaml and a specific multus image